Remotion/ObjectBinding/Web.Development.WebTesting.IntegrationTests/BocListAsGridControlObjectTest.cs

```diff
@@ -470,7 +470,9 @@ public void TestRowGetRowDropDownMenu ()
       var bocList = home.ListAsGrids().GetByLocalID("JobList_Normal");
       var row = bocList.GetRow(2);
       var dropDownMenu = row.GetDropDownMenu();
-      dropDownMenu.SelectItem("RowMenuItemCmd2");
+
+      var completionDetectionStrategy = new WebTestActionOptions { CompletionDetectionStrategy = new WxePostBackCompletionDetectionStrategy(1, TimeSpan.FromSeconds(2)) };
+      dropDownMenu.SelectItem("RowMenuItemCmd2", completionDetectionStrategy);
```

Comment:
Short explanation please about cause and effect. WXE-based operations in the web tests should already handle waiting for the returning postback so I'm confused why we need to a timeout?

Reply:
Our problem exists because the postback is not correctly waited for, as far as I understand the problem. There is in fact no timeout as far as I can see

Reply:
Nevermind, found the timeout. Then I don't know what's happening here :)

Reply:
-> Standup